src/large-sum/large-sum-utils.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
+  calculateSumCarryOver: arr => (arr.length * 9).toString().length,
+  createCalcArray: arr =>
```

Comment:
Also, this function. Walk me through what is going on here and why its needed.

Reply:
In this function, I am trying to convert each large number into an array of single digits, eg 1234 -> [1,2,3,4].  Initially, I was going to accomplish this by converting the large number into a string, split the string into individual characters, and then parse each character as an Integer, which would get me the result I wanted.  However, I ran into some issues since JavaScript doesn't handle large numbers well and loses precision.  JavaScript interprets the large number 37107287533902102798797998220837590246510135740250 as something like 3.71072875339021e+49.  This is why I ended up splitting the stringified large number on "e" and then selecting the first index, and since I didn't want to handle the decimal, I filtered it out. This isn't ideal, but it did give me sufficient precision to complete the problem.

Reply:
Thinking back, I probably could/should have reached for Large Number JavaScript library to handle this.

Reply:
> JavaScript interprets the large number 37107287533902102798797998220837590246510135740250 as something like 3.71072875339021e+49.  This is why I ended up splitting the stringified large number on "e" and then selecting the first index, and since I didn't want to handle the decimal, I filtered it out.

So how many digits were you keeping? 

When would your algorithm yield incorrect results?

Reply:
I am keeping as many as the stringified large number would capture, I guess this is dependant on how JavaScript interprets the number.  In the example, 3.71072875339021e+49 would keep 15 digits, [3,7,1,0,7,2,8,7,5,3,3,9,0,2,1]. As I was building this out, I saw that I was keeping between 14 - 18 digits.  I guess I am not 100% certain that this is going to be the result all the time.  So, if JavaScript interprets a large number as 1.234e+49, I would only get back 4 digits and this would lead to incorrect results.

Reply:
OK, so how would you fix that?

Reply:
I guess the best fix would be to take a step back and redefine the function to take an array of Large Number strings.  I would need to refactor the data array to be an array of strings and update my tests.  The resulting function would then be much cleaner and less hacky:
```
createCalcArray: arr =>
arr.map(ele =>
      ele
        .split("")
        .map(strInt => parseInt(strInt, 10))
    )
```

Reply:
Here is the PR to my fix: https://github.com/esack7/fluke-networks-project/pull/2